src/desktop/apps/article/__tests__/routes.test.js

```diff
@@ -82,7 +82,7 @@ describe("Article Routes", () => {
       })
     })
 
-    it("sets the correct jsonld", done => {
+    xit("sets the correct jsonld", done => {
```

Comment:
There is an in-progress ticket in Grow where @xtina-starr is fixing up these tests in Jest.